core/src/components/badge/test/hint/index.html

```diff
@@ -98,6 +98,66 @@
             </ion-avatar>
           </div>
         </ion-list>
+
+        <ion-list id="tab-button">
+          <ion-list-header>
+            <ion-label> Inside Tab Button </ion-label>
+          </ion-list-header>
+          <div class="ion-display-flex ion-align-items-center ion-justify-content-around ion-margin">
+            <ion-tab-button href="" tab="tab-one">
```

Comment:
All the `ion-tab-button` components need to be placed inside a `ion-tab-bar` for it to [render](https://ionicframework.com/docs/api/tab-button) correctly. The difference can be seen when viewing this [page](http://localhost:3333/src/components/tab-bar/test/basic) and compare it to this badge hint page.